code/src/components/ClearAllButton.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import swal from 'sweetalert';
+
+import { todos } from '../reducers/todos';
+import { SectionDeleteButton, StyledInput } from '../styledComponents/styled_components';
+
+export const ClearAllButton = () => {
+  const dispatch = useDispatch();
+  
+  const handleOnClick = () => {
+    swal({
```

Comment:
Really nice Sweet Alert! Goes very well with the app. I have not used one before, so it was very helpful to see how you have implemented it.